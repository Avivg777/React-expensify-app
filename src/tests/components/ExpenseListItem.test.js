import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../Components/ExpenseListItem/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('Should render a single expense list item correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>);

    expect(wrapper).toMatchSnapshot();
});