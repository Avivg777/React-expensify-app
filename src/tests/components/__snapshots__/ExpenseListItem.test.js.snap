// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a single expense list item correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    Amount: 
    150
  </p>
  <p>
    Created At: 
    0
  </p>
</div>
`;
